sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedLabeledPage.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    public class ExtractedLabeledPage
+    {
+        // Supervised
+        internal ExtractedLabeledPage(int pageNumber, List<ExtractedField> fields, PageResult_internal pageResult, ReadResult_internal readResult)
+        {
+            PageNumber = pageNumber;
+            Fields = ConvertFields(fields);
```

Comment:
`Fields = fields.AsReadOnly();`

Reply:
I think everywhere I'm calling ConvertFields() now, I'm actually converting the type.